sparkle/src/components/Markdown.tsx

```diff
@@ -144,7 +144,7 @@ export function Markdown({
                 className={classNames(
                   "s-w-full s-italic",
                   paragraphSize,
-                  textColor
```

Comment:
That won't be applied but only caught in the next Sparkle publish, good for you?